website/content/docs/internals/architecture.mdx

```diff
@@ -6,116 +6,84 @@ description: Learn about the internal architecture of Vault.
 
 # Architecture
 
-Vault is a complex system that has many different pieces. To help both users and
-developers of Vault build a mental model of how it works, this page documents
-the system architecture.
+Vault is an intricate system with numerous distinct components. This page details the system architecture and hopes to assist Vault users and developers to build a mental model while understanding the theory of operaton.
 
-~> **Advanced Topic!** This page covers technical details of Vault. You don't
-need to understand these details to effectively use Vault. The details are
-documented here for those who wish to learn about them without having to go
-spelunking through the source code. However, if you're an operator of Vault, we
-recommend learning about the architecture due to the importance of Vault in an
-environment.
+~> **Advanced Topic!** This page covers the technical details of Vault. The descriptions and elements contained within are for users that wish to learn about Vault without having to reference the source code. Although not required, we encourage all users and operators to review the provided information before using Vault due to its significance in an environment.
 
 # High-Level Overview
 
-A very high level overview of Vault looks like this:
+The diagram below illustrates the intricacies and distinct components of Vault.
 
 [![Architecture Overview](/img/layers.png)](/img/layers.png)
 
-Let's begin to break down this picture. There is a clear separation of
-components that are inside or outside of the security barrier. Only the storage
-backend and the HTTP API are outside, all other components are inside the
-barrier.
-
-The storage backend is untrusted and is used to durably store encrypted data.
-When the Vault server is started, it must be provided with a storage backend so
-that data is available across restarts. The HTTP API similarly must be started
-by the Vault server on start so that clients can interact with it.
-
-When a Vault server is started, it starts in a _sealed_ state. Before any
-operation can be performed on the Vault, it must be _unsealed_. This is done by
-providing the unseal keys. When the Vault is initialized, it generates an
-encryption key which is used to protect all Vault data. This key is protected by
-a root key. The root key is stored alongside all other Vault data, but is encrypted by yet another mechanism: the unseal key.
+Components are separated and considered as either inside or outside of the security barrier. The storage backend and HTTP API are outside of the security barrier, while all other components reside inside the barrier.
+
+When the Vault server starts, a storage backend is provided, making its data available across restarts. The storage backend, outside the barrier, stores encrypted data durably and is considered untrusted. The HTTP API must start similarly by the Vault server so that clients can interact with it.
```

Comment:
This part can re-write better?  In a nutshell, the way it works is...

The storage backend is outside of the barrier (therefore, untrusted), so Vault encrypts data before sending them to the storage backend. So, even if unauthorized attacker access the storage backend, the data is encrypted that there is no use until it gets decrypted. The _barrier_ is the Vault's encryption layer, so barrier encrypts and decrypts Vault data. Therefore, the storage backend provides durable data persistent layer where data is encrypted (secure), and make data available across server restarts.

Reply:
OK, let me try rewriting.